### ISSUE TITLE
Secure Connection Failed error

### ISSUE THREAD

**Issue by zdat** [ghacksuserjs/ghacks-user.js#895]
Was wondering which pref could be causing this error. Site seems to work on a stock firefox.

"An error occurred during a connection to X. Peer attempted old style (potentially vulnerable) handshake.
Error code: SSL_ERROR_UNSAFE_NEGOTIATION
The page you are trying to view cannot be shown because the authenticity of the received data could not be verified.
Please contact the website owners to inform them of this problem."

Thanks.

**Comment by geeknik** [comment]
Probably this one right here:  `user_pref("security.ssl.treat_unsafe_negotiation_as_broken", true);`

**Comment by zdat** [comment]
Thanks. The second one did it (1201). First one (1270) displays the warning on the padlock for sites that would normally be blocked by 1201.

**Comment by Thorin-Oakenpants** [comment]
- `1270` is about displaying a visual clue in the urlbar: it is not 100% reliable (e.g. subresources), but there is no harm in leaving it active in the user.js as true
- `1201` is the one that can break some sites: and it should be rare (everyone will get different mileage)

**Comment by bryce-lynch** [comment]
> `1201` is the one that can break some sites: and it should be rare

I just got the SSL error after clicking an eBay link (something like https://svcs.ebay.com/delstats/clk/rtm?url=....) and had to disable 1201. Weird.

**Comment by geeknik** [comment]
1201 breaks `tools.cisco.com` as well, they are aware of it, probably won't fix it as they can't understand why anyone would flip this pref on. 🤣
